src/cluster_legacy.c

```diff
@@ -6518,6 +6518,10 @@ int clusterCommandSpecial(client *c) {
             addReplyErrorFormat(c, "Invalid node address specified: %s:%s", (char *)c->argv[2]->ptr,
                                 (char *)c->argv[3]->ptr);
         } else {
+            sds client = catClientInfoString(sdsempty(), c, server.hide_user_data_from_log);
```

Comment:
The client info is very large and might complicate this output, maybe we can just print the client name and address? (we can also include the lib and version if that seems important)

Reply:
yean, adding a simple version seems to be a good idea.

Reply:
Will you add the short client info in this PR?

Reply:
yes, i will add it later, this indeed is a good idea.

Reply:
so, what fields should we put in the short client info? just client name and the address (and lib-name lib-version)?

Reply:
Is client name sensitive info? It is redacted in some cases, so maybe skip this one.

Lib name and version, addr and laddr? (including port)

In networking.c, we just log addr and laddr in some places, like `"Error accepting a client connection: %s (addr=%s laddr=%s)"`

Reply:
It was added here: a2e75a78b424a6faca4bba4b1bf8270b98407284

Reply:
i somehow think we can keep client name since these are mostly admin commands, a connection executing admin command, it is a admin user data. I am not sure about it, let's also use hide_user_data to control it.